### PR TITLE
[DOCS] Document the new rule execution status filter

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3486

Previews:
- [Manage detection rules](https://security-docs_3570.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-management.html): Made the following changes:
  - Refreshed screenshot of the Rules table
  - Added **Last response** filter to the list of filters under **Sort and filter the rules list**
  - Added a sentence on the **Last response** filter under **Check the current status of rules**
- [Monitor and troubleshoot rule executions](https://security-docs_3570.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-monitor.html): Refreshed screenshot of **Rule Monitoring** tab.

### CODE REVIEW

**Comment by nastasha-solomon** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/detections/images/monitor-table.png
Comment (joepeeples) [inline docs/detections/images/monitor-table.png:1]: Looks like the mouse pointer got captured on this one -- near the top above **Add Elastic rules**.

**Review by joepeeples — COMMENTED** [review]
One quick touch-up on the Rule Monitoring screenshot, otherwise LGTM!

File: docs/detections/images/monitor-table.png
Comment (nastasha-solomon) [reply docs/detections/images/monitor-table.png:1]: Oops, I thought I deleted that. I'll fix it now - thanks!

Comment (joepeeples) [reply docs/detections/images/monitor-table.png:1]: No problem! Once that image is fixed, this is probably good to merge -- we've had three writers working on this, and it's such a small change that we probably don't need a tech/dev/product review.

**Pull request merged**